### PR TITLE
Dod data

### PR DESCRIPTION
**Pull request by h1-the-swan**
This incorporates DoD data (in addition to NSF and NIH), comprising about 20,000 additional documents (grants). The data is in an updated CSV file that has not been checked in. This needs to be added to the elasticsearch index.

This PR adds DoD to the dropdown list of agencies (below NSF and NIH).

### CODE REVIEW

File: backend/app.py [hunk 1]
```diff
@@ -48,6 +46,17 @@
...
 word_vecs = None
 
+def load_word_vecs(path_to_model: Union[str, Path]) -> KeyedVectors:
+    # if the model is a '.txt' file, assume it is a text keyed vectors file
+    # otherwise, assume it is a full word2vec model file
+    # we'll probably want to change this
+    path_to_model = Path(path_to_model)
+    logger.info(f"loading similarity model from {path_to_model}")
+    if path_to_model.suffix == '.txt':
+        return KeyedVectors.load_word2vec_format(str(path_to_model), binary=False)
```
Comment (jessecoleman) [inline backend/app.py:56]: Is this just a different serialization format?

File: backend/ingest_data/index_grants.py [hunk 2]
```diff
@@ -62,7 +61,7 @@ class Grant(Document):
     # see also: Grant class in models.py
     grant_id = Keyword()
     agency = Keyword()
-    title = Text(fields={"raw": Keyword()})
+    title = Text(fields={"raw": Keyword()}, analyzer=aggressive_analyzer)
```
Comment (jessecoleman) [inline backend/ingest_data/index_grants.py:64]: What's this change doing? Just making the title/abstract have matching analyzers?

File: frontend/src/app/images/dod-logo.svg [hunk 3]
```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (jessecoleman) [inline frontend/src/app/images/dod-logo.svg:1]: Nice! I guess there were a few more things needed than what I listed, but you figured it out.

**Review by jessecoleman — APPROVED** [review]
Thanks for integrating the DOD data! I pulled it down and it looks good.

File: backend/app.py (hunk 1 above)
Comment (h1-the-swan) [reply backend/app.py:56]: It's a way of using the word vectors without loading the whole model. It doesn't allow continued training, but it's good if you just want to apply the trained vectors. See https://radimrehurek.com/gensim/models/keyedvectors.html

File: backend/ingest_data/index_grants.py (hunk 2 above)
Comment (h1-the-swan) [reply backend/ingest_data/index_grants.py:64]: Yes. We discussed this and you said there wasn't any reason the titles shouldn't have the same analyzer, that it wasn't intentional that they didn't match.

**Pull request merged**